### ISSUE TITLE
mitems on seq of tuples not working in VM

### ISSUE THREAD

**Issue by flyx** [nim-lang/Nim#5013]
```nim
static:
  var s = @[("baz", 42)]
  for i in s.mitems():
    if i[0] == "baz": i[1] = 3
  echo s
```

Output:

```
@[(Field0: baz, Field1: 42)]
```

Expected output (working without `static:`):

```
@[(Field0: baz, Field1: 3)] 
```

Works when child type of `seq` is an `int`.

**Comment by Parashurama** [comment]
This is fixed is latest devel with PR #6226

**Comment by Yardanico** [comment]
@Araq this is fixed in PR #6226 (with a test-case)
